src/createServer.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { router: userRouter } = require('./routes/user.route.js');
+const { router: expenseRouter } = require('./routes/expense.route.js');
+const { resetUsers } = require('./services/user.service.js');
+const { resetExpenses } = require('./services/expense.service.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
```

Comment:
Calling `resetUsers()` and `resetExpenses()` every time the server is created might not be ideal in a production environment, as it could lead to data loss. Ensure this behavior is intended, or consider moving these calls to a separate initialization script or a development-only environment.